.github/actions/setup-dfx/action.yml

```diff
@@ -4,32 +4,8 @@ description: Setup DFX
 runs:
   using: 'composite'
   steps:
-    - name: Cache DFX
-      uses: actions/cache@v3
-      with:
-        path: /usr/local/bin/dfx
-        key: dfx-cache-${{ env.dfx_version }}
```

Comment:
This cache wasn't doing anything. `dfxvm` will still download and install new `dfx`. The caching is leftover from the approach we were using before the `dfinity/setup-dfx` action was released.